benchmark/assert/deepequal-buffer.js

```diff
@@ -1,39 +1,62 @@
 'use strict';
 const common = require('../common.js');
 const assert = require('assert');
+
 const bench = common.createBenchmark(main, {
-  n: [1e3],
+  n: [1e5],
```

Comment:
Using different n is not really useful as far as I can tell. And as @refack pointed oout using 1e5 is still a low value.